docs/admin.rst

```diff
@@ -153,6 +153,41 @@ Server Version
 
 When debugging a problem it's important to verify the PostgREST version. At any time you can make a request to the running server and determine exactly which version is deployed. Look for the :code:`Server` HTTP response header, which contains the version number.
 
+.. _error_source:
+
+Error Source
+------------
+
+For the most part, error messages will come directly from the database with the same `format that PostgreSQL uses <https://www.postgresql.org/docs/current/plpgsql-errors-and-messages.html>`_, in other words, PostgREST will convert the ``MESSAGE``, ``DETAIL``, ``HINT`` and ``ERRCODE`` from the PostgreSQL error to JSON format and add an HTTP status code to the response (see :ref:`status_codes`). For instance, this is the error you will get when querying a nonexistent table:
```

Comment:
How about moving this whole doc to a dedicated "Errors" reference page?

We could change this section https://postgrest--430.org.readthedocs.build/en/430/api.html#http-status-codes to be there too.

Reply:
Looks great now!